src/scales/scale.time.js

```diff
@@ -175,8 +175,23 @@ module.exports = function(Chart) {
 		var ticks = [];
 		if (options.maxTicks) {
 			var stepSize = options.stepSize;
-			ticks.push(options.min !== undefined ? options.min : niceRange.min);
-			var cur = moment(niceRange.min);
+			var startTick = options.min !== undefined ? options.min : niceRange.min;
+			var seniorUnitStart = moment(startTick).add(1, options.seniorUnit).startOf(options.seniorUnit);
```

Comment:
These `options` are not user defined options. This is object constructed on line #347 and passed to `Chart.Ticks.generators.time` call and the inside this call passed to `generateTicks` function at line #231